### PR TITLE
20201201: add SRU verifications for GH: #575

### PR DESCRIPTION
**Pull request by TheRealFalcon**

### CODE REVIEW

File: bugs/gh-575.txt [hunk 1]
```diff
@@ -0,0 +1,246 @@
...
+
+=== Begin SRU Template ===
+[Impact]
+New functionality was added  allowing a root user to validate the userdata
+provided to the instance by running a new CLI option.
+
+[Test Case]
+
+There is an upstream test for this behavior:
+
```
Comment (lucasmoura) [inline bugs/gh-575.txt:11]: I think the test link is missed

**Review by lucasmoura — COMMENTED** [review]
Just a quick update to add the test link and LGTM

File: bugs/gh-575.txt [hunk 2]
```diff
@@ -0,0 +1,246 @@
...
+image_id=local:mature-rhino-snapshot
+['lxc', 'init', 'local:mature-rhino-snapshot', '--config', "user.user-data=runcmd:\n  - echo 'hi' > /var/tmp/test\n"]
+Launched instance: LXDInstance(name=present-tiger)
+.Deleting snapshot image created for this testrun: local:mature-rhino-snapshot
+
+
+======================== 2 passed in 126.29s (0:02:06) =========================
+
+[Regression Potential]
+None. It's a new command that doesn't run unless invoked.
```
Comment (OddBloke) [inline bugs/gh-575.txt:246]: This isn't a new command, it's a new switch to an existing command. We could potentially have regressed existing users of the `schema` subcommand if the modifications have introduced a bug that we haven't caught.

File: bugs/gh-575.txt (hunk 1 above)
Comment (TheRealFalcon) [reply bugs/gh-575.txt:11]: That's what I get for a last minute push at the end of the workday :wink:

**Pull request merged**